golden_response/staging/get_stat_ranking/crime_percapita.json

```diff
@@ -10,12 +10,12 @@
           },
           {
             "rank": 2,
-            "value": 0.185145406477198,
+            "value": 0.19114636642784,
```

Comment:
Crime data didn't change, perhaps the denominator we use changed sources/mmethods?

Reply:
Yes, as noted in the other comments, the choice of population value could have changed.